lighthouse-cli/bin.ts

```diff
@@ -281,6 +281,14 @@ function runLighthouse(url: string,
       return results;
     })
     .then((results: Results) => {
+      if (url !== results.url) {
```

Comment:
should we move this patch to runner.js so our other distributions get the log as well?